### ISSUE TITLE
Course list stuck in "Loading, please wait"

### ISSUE THREAD

**Issue by sigurdurb** [jupyter/nbgrader#1175]
What version of Jupyterhub are you guys using with the course_list extension? my tab for the course list is stuck in "Loading, Please wait..." and I see am getting a 404 error in the browser console. So no courses are loading. The path it is calling is /user/USERNAME/formgraders?_=1565965239559

nbgrader                  master at 16 august 2019
jupyterhub                0.8.1

@jhamrick

**Comment by rkdarst** [comment]
You should check the output from the single-user server for possible error messages - this is the most likely place to find the sign of the problem.  Where you find these depends on what JupyterHub spawner you use.

JupyterHub just passes it through to the single-user server.  As long as it's still running, there aren't many thing that can go wrong with JupyterHub.  Though it's not impossible.

**Comment by sigurdurb** [comment]
Yes I already looked but there is no error message, just this 404 line that has the same info as the console message.
404 GET /user/USERNAME/formgraders?_=1566222761680 (USERNAME@10.6.15.67) 9.60ms

**Comment by jhamrick** [comment]
I think this might be related to #1181. In which case, I will see about getting a fix for that ASAP (this weekend hopefully?)

**Comment by jhamrick** [comment]
Now that #1118 is fixed, are you still having a problem with this?

If there's still a problem, it possibly sounds like you don't have the server extension part of the course list extension activated as it's giving a 404. Can you try reactivating the course list extension and then restarting JupyterHub, and see if that helps?

```
jupyter nbextension enable --user course_list/main --section=tree
jupyter serverextension enable --user nbgrader.server_extensions.course_list
```

If it's still a problem after that too, can you paste the log output?
